website/frontend/src/tests/FormPayload/FormPayload.test.tsx

```diff
@@ -0,0 +1,500 @@
+import { Formik, Form } from 'formik';
+import { render, screen, within, fireEvent, act, waitFor, cleanup } from '@testing-library/react';
+
+import { FormValues, TestFormProps } from './types';
+import { INPUT_PAYLOAD_VALUES, INPUT_MANUAL_PAYLOAD, INIT_FORM_VALUES, INPUT_FILE_CONTENT } from './inputData';
+
+import { FILE_TYPES } from 'consts';
+import { getPreformattedText } from 'helpers';
+import { AlertProvider } from 'context/alert';
+import { FormPayload } from 'components/common/FormPayload';
+
+const TestFromPayload = ({ values, onSubmit }: TestFormProps) => (
+  <AlertProvider>
+    <Formik initialValues={INIT_FORM_VALUES} onSubmit={onSubmit}>
+      <Form data-testid="test-form">
+        <FormPayload name="payload" values={values} />
+      </Form>
+    </Formik>
+  </AlertProvider>
+);
+
+describe('from payload tests', () => {
+  const submitCallbackMock = jest.fn();
+
+  const submit = () => {
+    submitCallbackMock.mockReset();
+    fireEvent.submit(screen.getByTestId('test-form'));
+  };
+
+  const changeFieldValue = (element: Element | Node, value: string) =>
+    act(() => {
+      fireEvent.change(element, { target: { value } });
+    });
```

Comment:
I would rethink wrapping events in `act`, [source](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#wrapping-things-in-act-unnecessarily) and [fix](https://kentcdodds.com/blog/fix-the-not-wrapped-in-act-warning)